### ISSUE TITLE
add script to diagnose what blocked given domain

### ISSUE THREAD

**Issue by matkoniecz** [StevenBlack/hosts#116]
For example I discovered that some website is blocked what was not the expected result - it would be nice to have a built in script to tell me which list(s) included this rule

**Comment by StevenBlack** [comment]
I use `grep` :-)

**Comment by matkoniecz** [comment]
Thanks!

And that is why I open issue before starting to code immediately after thinking about something.
